addons/point_of_sale/static/tests/tours/helpers/ProductScreenTourMethods.js

```diff
@@ -211,19 +212,7 @@ export function scan_barcode(barcode) {
             content: `PoS model scan barcode '${barcode}'`,
             trigger: ".pos", // The element here does not really matter as long as it is present
             run: () => {
-                window.posmodel.env.services.barcode_reader.scan(barcode);
-            },
-        },
-    ];
-}
-export function scan_ean13_barcode(barcode) {
```

Comment:
Nice work! Could you move this function  to `point_of_sale/static/tests/tours/helpers/utils.js` and change the trigger to something that is not specific to pos, such as simply `body`, so that we could use this same function in kiosk also?